toolchain/systemtap.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2017 IBM
+# Author: Pooja B Surya <pooja@linux.vnet.ibm.com>
+# Based on code by Anton Blanchard <anton@samba.org>
+
+# https://github.com/autotest/autotest-client-tests/tree/master/systemtap
+
+import os
+
+from avocado import Test
+from avocado import main
+from avocado.utils import build
+from avocado.utils import process
+from avocado.utils import git
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Systemtap(Test):
+
+    def setUp(self):
+        smm = SoftwareManager()
+        packages = ['make', 'gcc', 'systemtap', 'systemtap-runtime',
+                    'elfutils', 'kernel-devel', 'kernel-debug',
+                    'kernel-debuginfo', 'kernel-debug-debuginfo', 'dejagnu',
```

Comment:
i think for ubuntu kernel-debuginfo package name  will be different, can you please check